.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
+venv/
+db.sqlite3
+**__pycache__/
```

Comment:
missed blank line

Reply:
![Screenshot 2024-05-25 122111](https://github.com/mate-academy/py-movie-api/assets/108980023/bdc7de8f-dc9b-483b-963c-c527de65693a)
deleted by the previous commit, added by the next. I'm inattentive.